src/services/dbSyncer.ts

```diff
@@ -0,0 +1,130 @@
+import * as Knex from 'knex';
```

Comment:
Filename could still be `dbSync`. Haha `dbSyncer`?

Reply:
Naming is always difficult :slightly_smiling_face: